ipatests/test_webui/test_subid.py

```diff
@@ -125,3 +127,20 @@ def test_user_subid_add(self):
         self.assert_no_error_dialog()
         after_count = self.get_rows()
         assert len(before_count) < len(after_count)
+
+    @screenshot
+    def test_subid_range_deletion_not_allowed(self):
+        """
+        Test to check that subid range delete is not
+        allowed from WebUI i.e Delete button is not available.
+        """
+        self.init_app()
+        self.navigate_to_entity('subid', facet='search')
+        self.facet_button_click('add')
+        self.select_combobox('ipaowner', 'admin')
+        self.dialog_button_click('add')
+        self.wait(0.3)
+        self.assert_no_error_dialog()
+        self.get_field_checked('ipauniqueid')
+        with pytest.raises(selenium.common.exceptions.NosuchElementExcpetion):
```

Comment:
You have imported `NoSuchElementException` already, so the correct usage would be 
```
with pytest.raises(NosuchElementExcpetion):
...
```